src/coreclr/src/nativeaot/System.Private.CoreLib/src/Internal/Console.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Text;
+using Microsoft.Win32.SafeHandles;
+
+namespace Internal
+{
+    //
+    // Simple limited console class for internal printf-style debugging in System.Private.CoreLib
+    // and low-level tests that want to call System.Private.CoreLib directly
+    //
+
+    public static class Console
+    {
+#if TARGET_WINDOWS
+        private static readonly SafeFileHandle _outputHandle =
+            new SafeFileHandle(Interop.Kernel32.GetStdHandle(Interop.Kernel32.HandleTypes.STD_OUTPUT_HANDLE), ownsHandle: false);
```

Comment:
It may be nice to avoid the SafeHandle, and use `WriteFile_IntPtr.cs` to print the message instead.